activerecord/lib/active_record/autosave_association.rb

```diff
@@ -280,7 +280,7 @@ def associated_records_to_validate_or_save(association, new_record, autosave)
       end
 
       # go through nested autosave associations that are loaded in memory (without loading
-      # any new ones), and return true if is changed for autosave
+      # any new ones), and return true if any changed for autosave.
```

Comment:
While we're here, can we capitalize `Go` on the previous line and add `are` after `any` here?

Reply:
Done! I've also added an explanation about the infinite loop prevention.

Reply:
@gmcgibbon I've made the requested changes :)